package.json

```diff
@@ -79,7 +81,22 @@
         ],
         "configuration": "./ansi.language-configuration.json"
       }
-    ]
+    ],
+    "configuration": {
+      "title": "ANSI Colors",
+      "properties": {
+        "iliazeus.vscode-ansi.autoPreview": {
+          "type": "boolean",
+          "default": true,
+          "description": "Automatically open the ANSI Text preview without having to use the keyboard shortcut to open it."
+        },
+        "iliazeus.vscode-ansi.autoPreviewToSide": {
+          "type": "boolean",
+          "default": false,
+          "description": "If Auto Preview is set to True: Automatically open the ANSI Text Preview to the side. Warning: This can cause undesirable behavior!"
```

Comment:
Can you please explain what this undesirable behavior is? Maybe we can fix it, or just don't include this specific option.